app/src/main/java/io/webthings/app/utils/NavigationHost.kt

```diff
@@ -1,17 +1,18 @@
-package io.webthings.app.utils
+package io.webthings.app.ui
```

Comment:
```suggestion
package io.webthings.app.utils
```
The package name should match the directory structure.